packages/eslint-plugin/src/configs/base.json

```diff
@@ -0,0 +1,13 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "sourceType": "module"
+  },
+  "plugins": ["@typescript-eslint"],
+  "rules": {
+    "camelcase": "off",
+    "indent": "off",
+    "no-array-constructor": "off",
+    "no-unused-vars": "off"
```

Comment:
```suggestion
    "no-unused-vars": "off",
    "no-useless-constructor": "off"
```

We ship our own `@typescript-eslint/no-useless-constructor` and the built-in eslint one may choke on some syntactical constructs.

#### Related

- https://github.com/typescript-eslint/typescript-eslint/pull/167
- https://github.com/typescript-eslint/typescript-eslint/pull/329#issuecomment-469688633

Reply:
Sounds reasonable. Solved 😄